test/files/get_available_file_path_test.dart

```diff
@@ -0,0 +1,152 @@
+import 'dart:io';
+import 'package:fluffychat/utils/platform_infos.dart';
+import 'package:fluffychat/utils/storage_directory_utils.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Future<File> createMockFile(String filePath) async {
+  final file = File('$folderTestName/$filePath');
+  await file.create(recursive: true);
+  return file;
+}
+
+const folderTestName = 'generated';
+
+void main() async {
```

Comment:
how about the case:
- folder have: `file.pdf`, `file (1).pdf`, `file (3).pdf`, `file (5).pdf` and download new file is `file.pdf`

Reply:
then new file have name file (2).pdf. update the test to add this case